### PR TITLE
feat: add bug report button

### PR DESCRIPTION
**Pull request by tilmx**
Adds a bug report button in the chrome that leads to the Github Issues page with bugs. This might be an easy approach for users during the Alpha/Beta phase to immediately get the right location for sending bugs.

Should be squashed into `feat: add bug report button`

### CODE REVIEW

File: src/container/chrome/chrome-container.tsx [hunk 1]
```diff
@@ -70,6 +71,14 @@ export const ChromeContainer = MobxReact.inject('store')(
 						title={project ? project.getName() : 'Alva'}
 					/>
 				)}
+				<BugReport
+					title="Found a bug?"
+					onClick={() => {
+						Electron.shell.openExternal(
```
Comment (marionebl) [inline src/container/chrome/chrome-container.tsx]: We are in the process of moving all `Electron` dependencies to the main process. We should move this call to `electron/main.js` and send an event to trigger it instead

Comment (tilmx) [reply src/container/chrome/chrome-container.tsx]: Sounds like a good idea, but how would I do that @marionebl?

Comment (tilmx) [reply src/container/chrome/chrome-container.tsx]: Got an idea. Trying it out now.

**Comment by tilmx** [comment]
Moved it to the main process @marionebl

File: src/components/bug-report/index.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import styled from 'styled-components';
+
+export interface BugReportProps {
+	onClick?: React.MouseEventHandler<HTMLElement>;
+	title: string;
+}
+
+const StyledBugReport = styled.div`
+	justify-self: right;
+	margin-right: -${getSpace(SpaceSize.XXL) * 3 - getSpace(SpaceSize.L)}px; // align to top right corner
```
Comment (LKuechler) [inline src/components/bug-report/index.tsx]: The padding should be removed in the `chrome` component

File: src/container/chrome/chrome-container.tsx [hunk 3]
```diff
@@ -70,6 +73,17 @@ export const ChromeContainer = MobxReact.inject('store')(
 						title={project ? project.getName() : 'Alva'}
 					/>
 				)}
+				<BugReport
+					title="Found a bug?"
+					onClick={() => {
+						console.log('chrome-container started');
```
Comment (LKuechler) [inline src/container/chrome/chrome-container.tsx]: This is just debug code right? ^^

File: src/components/bug-report/index.tsx (hunk 2 above)
Comment (tilmx) [reply src/components/bug-report/index.tsx]: Yes, but the padding leads to the fact, that the "< Pages" indicator is under the 3 mac close icons. Or if we only set the right padding differently, the center title is not properly aligned (a bit too right). That’s why I chose that solution.

File: src/container/chrome/chrome-container.tsx (hunk 3 above)
Comment (tilmx) [reply src/container/chrome/chrome-container.tsx]: Yes. Good spot!

**Pull request merged**